app/containers/message/PhotoModal.js

```diff
@@ -51,6 +54,44 @@ export default class PhotoModal extends React.PureComponent {
 		window: PropTypes.object
 	}
 
+	saveToCameraRoll = async(image) => {
+		if (Platform.OS === 'android') {
+			if (await this.permission()) {
+				RNFetchBlob
+					.config({
+						fileCache: true,
+						appendExt: 'jpg'
+					})
+					.fetch('GET', image)
+					.then((res) => {
+						CameraRoll.saveToCameraRoll(res.path())
+							.then(() => Alert.alert(I18n.t('Save_To_Camera_Success'), I18n.t('Save_To_Camera_Success_Message')))
+							.catch(err => console.error('err:', err));
+					});
+			} else {
+				Alert.alert(I18n.t('Save_To_Camera_Error'), I18n.t('Save_To_Camera_Error_Message'));
```

Comment:
can you use this https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/48b53e852ac35a4b0eded5d75ee24166d06c236d/app/utils/info.js#L5https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/48b53e852ac35a4b0eded5d75ee24166d06c236d/app/utils/info.js#L5 it helps to maintain look and feel of all error messages